### PR TITLE
update README

### PR DESCRIPTION
**Pull request by sebastiendavid**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,19 @@
 # Join the team
 
-Get your name in top of the list with the highest score on https://alkemics.github.io/sf2/ and send us a mail on steve@alkemics.com with the URL.
+Our developers are the best at Street Fighter, here our high score page:
```
Comment (adlenafane) [inline README.md]: `here is`

File: README.md [hunk 2]
```diff
@@ -1,3 +1,19 @@
...
+
+https://alkemics.github.io/sf2/
+
+To apply, please email `steve@alkemics.com` with "**Hire me, I'm the best at Street Fighter**" as the subject, and include a link to our high scores page in the body, and hopefully a link that leads to our page where you are the one being the best!
+
+To be clear, solving this puzzle involves finding a way to trick our high score page buggy code and put your name on top of the list. The body of your email should look something like this:
```
Comment (adlenafane) [inline README.md]: Does Trello provide more information like this?

**Comment by adlenafane** [comment]
Couple of comments LGTM otherwise

File: README.md (hunk 2 above)
Comment (birdy-) [inline README.md]: Nop

Comment (sebastiendavid) [inline README.md]: yep it's Trello explanation customized by me

Comment (birdy-) [inline README.md]: Remove the `To be clear...`

```
To apply, please email `steve@alkemics.com` with "**Hire me, I'm the best at Street Fighter**" as the subject, and include a link to our high scores page with you on top of the list !
```
Something like this ? Keep it short.

**Pull request merged**